src/views/Header.js

```diff
@@ -306,30 +326,26 @@ const styles = StyleSheet.create({
     elevation: 4,
   },
   appBar: {
-    flex: 1
+    flex: 1,
+    position: 'relative',
   },
-  item: {
+  header: {
     flexDirection: 'row',
-    alignItems: 'center',
   },
-  title: {
-    bottom: 0,
-    left: 40,
-    position: 'absolute',
-    right: 40,
-    top: 0,
+  item: {
+    justifyContent: 'center',
   },
+  title: Platform.OS === 'android'
+    ? {
+      flex: 1,
+      alignItems: 'flex-start',
+    }
+    : null,
   left: {
-    bottom: 0,
-    left: 0,
-    position: 'absolute',
-    top: 0,
+    alignItems: 'flex-start',
```

Comment:
Nope, `alignItems` aligns vertically when flex is `row`. In `column`, it aligns horizontally (see screenshots), so that left children (`renderLeftComponent()`) is always to the left of its Animated container.